query/filter/logical_test.go

```diff
@@ -40,43 +40,38 @@ func TestLogicalToSearch(t *testing.T) {
 	}
 
 	js := []byte(`{"f1": 10}`)
-	testLogicalSearch(t, js, factory, []string{"f1:=10"})
+	testLogicalSearch(t, js, factory, "f1:=10")
```

Comment:
This test demonstrates changes to search filter serialization for the search backend